### ISSUE TITLE
Feature Request: packer EBS builder - share encrypted AMIs with additional accounts (new AWS SDK functionality) - duplicate

### ISSUE THREAD

**Issue by ajhool** [hashicorp/packer#9838]
There is an existing issue with 14 thumbs up that was closed as stale. Per the StaleBot request, this is a duplicate of that issue with some updated info:

https://github.com/hashicorp/packer/issues/7681

#### Community Note

Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request.
If you are interested in working on this issue or have submitted a pull request, please leave a comment.

#### Description

AWS recently announced the ability to share encrypted AMIs across accounts using a single SDK command.

https://aws.amazon.com/about-aws/whats-new/2019/05/share-encrypted-amis-across-accounts-to-launch-instances-in-a-single-step/

This would be a great feature to incorporate into packer since it already has the ability to share non-encrypted AMIs with supplied account numbers. It could leverage the same template format since I believe packer can already share unencrypted AMIs.

#### Use Case(s)

Create a packer-built AMI that automatically shares an encrypted AMI with our customers + KMS key access  with our customers. Our customers are then free to launch ec2 instances using the encrypted AMIs. Accounts who are nott our customers *don't* have automatic access to this key..

Single-region strategy to start.

Note: the documentation does say that the new ami will maintain the same encryption status as the previous AMI. That might be one way to make this work; however, if we are also building the base ami in the same packer config, then it would be nice to have a way to do this.

#### Potential configuration

```
ami_users: [customer_a_account_id, customer_b_account_id],
kms_key_id: "builder-account-kms-key-id"
```

#### Potential References

https://aws.amazon.com/about-aws/whats-new/2019/05/share-encrypted-amis-across-accounts-to-launch-instances-in-a-single-step/

**Comment by nywilken** [comment]
Hi there, thanks for opening up the issue again. I'm not sure if we will have a chance to look into this feature soon, but we would gladly help review and give guidance on a pull-request if you or a member of the Packer community is interested in contributing to the code base.

**Comment by SwampDragons** [comment]
The issue was locked by the bot because it had been resolved and closed in #7707. 

What version of Packer are you on? This should have been implemented in v1.4.2

**Comment by nywilken** [comment]
Hello there @ajhool since this feature has been implemented already. I am going to close this issue. If you haven't already please check that you are using a version of Packer greater than v1.4.1 in order to use this feature. If you are running a later version and running into issues please let us know so that we can investigate further. Cheers!
